04 - Guides, Workflows, & Courses/for Theme Designers.md

```diff
@@ -37,6 +37,7 @@ This note collects resources and guides for beginner and expert theme designers
 ## Tools
 - [**Obsidian Theme Template**](https://github.com/obsidian-community/obsidian-theme-template) A GitHub repository template with everything you need to get started. Click `Use this template` to get a ready-to-go theme repository.
 - [**Theme Dev Vault**](https://github.com/obsidian-community/theme-dev-vault) An obsidian vault to aid in theme development
+- [Theme Design Utilities Plugin](https://github.com/chrisgrieser/obsidian-theme-design-utilities)
```

Comment:
Thanks! Is there a particular reason for linking directly to the GitHub repo? I ask because I see there's a note for this already, so wondered whether it might link to that instead?

https://publish.obsidian.md/hub/02+-+Community+Expansions/02.05+All+Community+Expansions/Plugins/obsidian-theme-design-utilities

Reply:
oh you are right, simply forgot.

Reply:
updated